homeworks/01-bubble-sort/src/components/bubbleGrid.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { CSSProperties } from "react";
+import "./bubbleGrid.css";
+import "./bubble.css";
+
+const heightMultiplier = 5;
+
+type BubbleGridProps = {
+    bubblesArray: number[]
+}
+
+export const BubbleGrid: React.FC<BubbleGridProps> = ({ bubblesArray }) => {
+    const bubbles = Object.values(bubblesArray).map((bubble: number, index: number) => (
```

Comment:
Why do you need `Object.values(bubblesArray)`? Why not just `bubblesArray.map()`?

Also, you don't need to define types here `map((bubble: number, index: number) =>`. It should be inferred automatically if you have a correct type for  `bubblesArray `.